test/fixtures/compound5.css

```diff
@@ -0,0 +1,9 @@
+
+#foo ul li {
+	--color: #ffffff;
+	color: var(--color);
+}
+
+#foo.bar li {
+	--color: #000000;
+}
```

Comment:
This seems like a known-limitation that should be in the output. Let's add a comment.

(maybe)